yarn.lock

```diff
@@ -106,11 +106,35 @@
     "@algolia/logger-common" "4.14.3"
     "@algolia/requester-common" "4.14.3"
 
-"@babel/parser@^7.16.4":
+"@apidevtools/json-schema-ref-parser@^9.0.3":
```

Comment:
I am not sure why, but the `yarn.lock` "exploded" in size? I have also delete the `package.lock.json` from the `firebaseFunctions` as since we are using `yarn` and not `npm`, but I am not sure whether that is correct, any thoughts?